mk/PcapPlusPlus.mk.common

```diff
@@ -15,3 +15,6 @@ PCAPPP_POST_BUILD :=
 # build flags
 PCAPPP_BUILD_FLAGS :=
 
+ifdef PCAPPP_ENABLE_CPP_FEATURE_DETECTION

```

Comment:
I don't see anywhere in the code where you use the `-DPCAPPP_CPP_FEATURE_DETECTION` flag

Reply:
@seladb This is the reserve of macro definition for future update - automatic c++11 feature recognition and support. I will consider **build up on** strategy for them where features are integrated and turned on only where they are applicable and they are not substituted or reimplemented in older version (uglier code but no pain with backwards compatibility).